src/components/Buttons/TurnButton.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+TurnButton.propTypes = {
+  onClick: PropTypes.func,
+  buttonText: PropTypes.string.isRequired,
+}
+
+export default function TurnButton({ arrowSymbol, buttonText, onClick }) {
+  return (
+    <ButtonStyled onClick={onClick}>
+      <p>{buttonText}</p>
+      <p>{arrowSymbol}</p>
+    </ButtonStyled>
+  )
+}
+
+const ButtonStyled = styled.button`
```

Comment:
I think the other buttons have similar styles. It might be useful to make a Button component that you can reuse. If you need to change styles you can use `styled(Button)` to change some styles.

Reply:
I tried reusing my PageButton component, but couldn't implement a line break, so the arrow symbol would be under the text, but I'll try to find a solution for this.